app/utils/scripts/mangaFileToImageDir.py

```diff
@@ -35,7 +36,10 @@ def mangaFileToImageDir(filepath: str):
         str: Path to directory of images.
     """
     extractPath, extension = splitext(filepath)
-    cachePath = f"./poricom_cache/{basename(extractPath)}"
+    if platform == "linux":
+        cachePath = f"/tmp/poricom_cache/{basename(extractPath)}"
+    else:
+        cachePath = f"./poricom_cache/{basename(extractPath)}"
```

Comment:
Instead of checking the current platform every time, create a variable `PORICOM_CACHE` in `utils/constants` and use it to get the correct path.

Reply:
Variables: `PLATFORM` and `PORICOM_CACHE` created in `utils/constants`